src/main/java/com/exadel/discountwebapp/location/service/LocationService.java

```diff
@@ -3,29 +3,28 @@
 import com.exadel.discountwebapp.exception.exception.client.EntityNotFoundException;
 import com.exadel.discountwebapp.filter.SpecificationBuilder;
 import com.exadel.discountwebapp.location.entity.Location;
-import com.exadel.discountwebapp.location.mapper.CityMapper;
 import com.exadel.discountwebapp.location.mapper.LocationMapper;
+import com.exadel.discountwebapp.location.projections.CityView;
 import com.exadel.discountwebapp.location.repository.LocationRepository;
-import com.exadel.discountwebapp.location.vo.city.CityResponseVO;
 import com.exadel.discountwebapp.location.vo.location.LocationRequestVO;
 import com.exadel.discountwebapp.location.vo.location.LocationResponseVO;
 import lombok.RequiredArgsConstructor;
+import org.codehaus.plexus.util.StringUtils;
```

Comment:
wrong library, let's use org.apache.commons.lang3.StringUtils;

Reply:
Changed to good one)